compiler/parser.mly

```diff
@@ -116,6 +117,24 @@ list:
   | expr                                  { [$1] }
   | list COMMA expr                       { $3 :: $1 }
 
+
+/* Distributions */
+dist_func:
+  | LCAR expr COMMA expr DDELIM expr VBAR
+    { {
+      min = $2;
+      max = $4;
+      dist_func = $6;
```

Comment:
@akalicki this will work, at least i haven't encountered any instances in which it doesn't. See check_dist in analyzer if you want to see how we are dealing w/ this.